src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        return Colors.values()[new Random().nextInt(Colors.values().length)].getColor();
```

Comment:
to improve the ColorSupplier class we can extract the creation of the Random object and store it as a field to avoid unnecessary object creation on every method call.